mxfusion/components/distributions/random_gen.py

```diff
@@ -83,6 +91,19 @@ def sample_multinomial(data, get_prob=True, dtype='int32', F=None):
         return F.random.multinomial(
             data=data, get_prob=get_prob, dtype=dtype)
 
+    @staticmethod
+    def sample_bernoulli(prob_true=0.5, dtype='bool', F=None):
```

Comment:
Why do you pass in prob_true if you don't actually use it?

Reply:
that's a bug! good catch